src/models/group.js

```diff
@@ -0,0 +1,32 @@
+export default (sequelize, DataTypes) => {
+  const Group = sequelize.define('group', {
+    id: {
+      type: DataTypes.UUID,
+      primaryKey: true,
+      defaultValue: DataTypes.UUIDV4
+    },
+    type: {
+      type: DataTypes.ENUM,
+      values: [
+        'PAIR',
+        'GROUP'
+      ],
+      allowNull: false
+    },
+    discount: {
+      type: DataTypes.FLOAT,
+      allowNull: false
+    },
+    expiresAt: {
+      type: DataTypes.DATE,
+      allowNull: false
+    }
```

Comment:
Let's add "minParticipants" and "maxParticipants". max is not required.

Reply:
Ok Done